assets/src/infrastructure/ui/inputs/dateTime/DateTimeRangePicker/style.scss

```diff
@@ -1,6 +1,16 @@
 @import 'application/ui/styles/mixins/screen-reader.scss';
 
 .ee-date-time-range-picker.ee-date-time-range-picker {
+	.react-calendar {
+		&__year-view {
+			.react-calendar {
+				&__tile {
+					padding: var(--ee-padding-small) var(--ee-padding-smaller);
```

Comment:
![image](https://user-images.githubusercontent.com/1477453/88697405-2ee8a500-d10d-11ea-8293-cfbaf00b4371.png)

Reply:
![image](https://user-images.githubusercontent.com/1477453/88697427-37d97680-d10d-11ea-8094-a9ef99b7bcb0.png)

Reply:
![image](https://user-images.githubusercontent.com/1477453/88697467-46279280-d10d-11ea-9366-b41a872adbf5.png)

Reply:
the month picker looks MUCH better